GSA/digitalgov.gov
# New Event Page URLs

# https://github.com/GSA/digitalgov.gov/issues/448

## What was changed?
Our event pages were changed to live at `/event/year/month/day/slug/`.
Formerly, they lived at `/event/slug/`, and that was causing a problem with reoccurring events.

Also `/event/` should now redirect to `/events/` 🎉 

## How to test

1. Go to any of the existing event pages on the [Preview URL](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/event-urls/events/)
2. Grab the end of the URL on an event page `/event/2018/04/18/dap-learning-series-creating-your-own-campaign-url-builder/`
3. Remove the date from the URL string. This will replicate the OLD url pattern: `/event/dap-learning-series-creating-your-own-campaign-url-builder/`
4. That URL should redirect to the NEW url with the date

**Here are a few examples links that should all redirect to the new URL with dates:**
- https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/event-urls/event/dap-learning-series-creating-your-own-campaign-url-builder/
- https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/event-urls/event/dap-learning-series-analytics-accessibility/
- https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/event-urls/event/dap-learning-series-writing-custom-javascript-for-tag-managers/
- https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/event-urls/event/a-deep-dive-into-top-tasks-with-gerry-mcgovern/
